Dockerfile

```diff
@@ -1,5 +1,7 @@
 FROM ruby:2.7-slim
 
```

Comment:
Is there a particular reason we don't use say `2.7-alpine` instead?  Based upon image scans, `2.7-slim` contains more vulnerabilities than `2.7-alpine`:

#### 2.7-slim scan

<img width="1602" alt="Screen Shot 2021-07-29 at 2 00 07 PM" src="https://user-images.githubusercontent.com/1253462/127565155-f7138737-b7a1-45ed-88ea-7ef5b59669c0.png">

#### 2.7-alpine

<img width="1584" alt="Screen Shot 2021-07-29 at 1 53 21 PM" src="https://user-images.githubusercontent.com/1253462/127565420-16b03c16-eeac-4fc3-ba89-84953fa4f724.png">

Reply:
Using alpine. Thanks!